release/RELEASE.md

```diff
@@ -4,28 +4,62 @@
    file and adds a new header and label for the new version in
    `docs/source/support/release-notes.rst` (see previous releases as examples).
    Release notes should be cut and pasted under the new header from `unreleased.rst`.
-   Each change outlined in `unreleased.rst` is preceded by the section to which it belongs: create one entry per section and add all pertaining items (without the section tag) to the release notes.
+   Each change outlined in `unreleased.rst` is preceded by the section to
+   which it belongs: create one entry per section and add all pertaining
+   items (without the section tag) to the release notes.
    It is important that the PR only changes `VERSION`, `release-notes.rst` and `unreleased.rst`.
-1. "Squash and merge" the PR.
+   Note that `unreleased.rst` and `release-notes.rst` must be modified even if
+   there have been no changes that have been added to the release notes so far.
+1. Merge the PR.
 1. Once CI has passed for the corresponding master build, the release should be
-   available on bintray and GitHub, as well as properly tagged.
-1. Activate the new version with `da use VERSION`. Note that it will
-   not be picked up by `da upgrade` at this point.
-1. Run through the manual test plan described in https://docs.google.com/document/d/16amcy7bQodXSHjEmKhAUiaPf6O92gUbch1OyixDEvSM/edit?ts=5ca5be00.
+   available on Bintray, Maven Central and GitHub and have a Git tag.
+   Maven central has a slight delay of around 20 minutes until the
+   new version is visible.
+1. Run through the following test plan on Linux or MacOS:
 
-   The test plan currently still targets the old `da` assistant. We
-   will migrate the test plan fully to the new assistant soon but for
-   now, there is a shorter test plan for the new assistant that
-   you should run on Windows:
+   1. Install the SDK using `curl -sSL https://get.daml.com/ | sh -s $SDK_VERSION`.
+   1. Run `daml version --assistant=yes` and verify that `$SDK_VERSION` is the 
+      assistant version and the default version for new projects.
+   1. Create a new project with `daml new quickstart quickstart-java`
+      and switch to it using `cd quickstart`.
+   1. Run `daml start`. Your browser should be opened automatically at
+      `http://localhost:7500`. Login as `Alice` and verify that there is
+      1 contract and 3 templates. Close the tab and kill `daml
+      start` using `Ctrl-C`.
+   1. Run `daml build -o .daml/dist/quickstart.dar`.
```

Comment:
Is the `-o .daml/dist/quickstart.dar` necessary here, given that this is the default location?

Reply:
Good point, removed.